openpype/plugins/inventory/remove_and_load.py

```diff
@@ -0,0 +1,43 @@
+from openpype.pipeline import InventoryAction
+from openpype.pipeline.legacy_io import Session
+from openpype.pipeline.load.plugins import discover_loader_plugins
+from openpype.pipeline.load.utils import (
+    get_loader_identifier,
+    remove_container,
+    load_container,
+)
+from openpype.client import get_representation_by_id
+
+
+class RemoveAndLoad(InventoryAction):
+    """Delete inventory item and reload it."""
+
+    label = "Remove and load"
+    icon = "refresh"
+
+    def process(self, containers):
+        for container in containers:
+            project_name = Session.get("AVALON_PROJECT")
+
+            # Get loader
+            loader_name = container["loader"]
+            for plugin in discover_loader_plugins(project_name=project_name):
+                if get_loader_identifier(plugin) == loader_name:
+                    loader = plugin
+                    break
+            else:
+                raise RuntimeError(
+                    "Failed to get loader, can't remove and load container"
+                )
+
+            # Get representation
+            representation = get_representation_by_id(
+                project_name, container["representation"]
+            )
+            assert representation, "Representation not found"
```

Comment:
Not sure if it should raise an exception?

Reply:
I think this inventory action should not proceed if `representation` is `None`, because it would manage to remove the container, then fail to reload it. This is why I chose to raise an exception. Eventually I could simply print and return but I'm not too sure if that would be better.

Reply:
Just continue before it's removed?

Reply:
I believe he means to do the assert after the `remove_container`

Reply:
That would be problematic because if the load cannot be performed then we shouldn't remove in my opinion. That's why I chose to assert before remove.

Reply:
I believe @iLLiCiTiT just means to not raise an assertion directly but to continue with the remainder of the containers and skipping the invalid ones with e.g. a `self.log.warning()` message as to why it's skipped that particular one.

So it would be like:
```suggestion
            if not representation:
                self.log.warning(
                    "Skipping remove and load because representation"
                    " id is not found in database: '{}'".format(container["representation"])
```

Reply:
Oh! I think I understand now, thanks you :smile: